examples/12-fluids/03-fluids_isosurface.py

```diff
@@ -85,16 +85,16 @@
 
 vec_iso_values = [-153.6, -100.0, -50.0, 50.0, 100.0, 150.0, 200.0, 300.0, 361.8]
 
-iso_surfaces = dpf.operators.mesh.iso_surfaces(field=P_S[0], mesh=whole_mesh, slice_surfaces=True,
-                                              vector_iso_values=vec_iso_values).eval()
-
-for i in range(len(iso_surfaces)):
-    P_S_step = dpf.Field(location=dpf.locations.overall, nature=dpf.common.natures.scalar)
-    P_S_step.append(vec_iso_values[i], i)
-    P_S_step.name = "static pressure"
-    P_S_step.unit = "Pa"
+iso_surfaces_meshes = dpf.operators.mesh.iso_surfaces(field=P_S[0], mesh=whole_mesh, slice_surfaces=True,
+                                              vector_iso_values=vec_iso_values).outputs.meshes()
+
+iso_surfaces_fields = dpf.operators.mesh.iso_surfaces(field=P_S[0], mesh=whole_mesh, slice_surfaces=True,
+                                              vector_iso_values=vec_iso_values).outputs.fields_container()
```

Comment:
With this you are evaluating the operator again, which can be very costly for large models (we shouldn't put it in an example). Instead, we should instantiate the operator once and then get the two outputs